linodecli/baked/response.py

```diff
@@ -0,0 +1,248 @@
+    :param response: The response we're checking
+    :type response: openapi3.Response
+    """
+    return (
+        response.schema.properties is not None
+        and len(response.schema.properties) == 4
+        and all(
+            c in response.schema.properties
+            for c in ("pages", "page", "results", "data")
+        )
+    )
+
+
+class OpenAPIResponseAttr:
+    """
+    Represents a single attribute of an API response as defined by the OpenAPI spec.
+    This class is given the schema node from the spec and parses out its own information
+    from it.
+    """
+
+    def __init__(self, name, schema, prefix=None):
+        """
+        :param name: The key that held this schema in the properties list, representing
+                     its name in a response.
+        :type name: str
+        :param schema: The schema this attribute will represent
+        :type schema: openapi3.Schema
+        :param prefix: The json path style prefix (dot notation) to this schema
+                       in the response object
+        :type prefix: str
+        """
+        #: The name of this attribute, which is the full json path to it within the schema
+        self.name = name if prefix is None else prefix + "." + name
+
+        #: TODO: Not sure what this is for
+        self.value = None
+
+        #: If this attribute is filterable in GET requests
+        self.filterable = schema.extensions.get("linode-filterable")
+
+        #: The description of this argument, for help display.  Only used for filterable attributes.
+        self.description = (
+            schema.description.split(".")[0] if schema.description else ""
+        )
+
+        #: No response model fields are required. This is only used for filterable attributes.
+        self.required = False
+
+        #: If this argument is Read Only
+        self.read_only = schema.readOnly
+
+        #: If this attribute should be displayed by default, and where in the output table
+        #: it should be displayed
+        self.display = schema.extensions.get("linode-cli-display") or 0
+
+        #: The name of the column header for this attribute.  This is the schema's name
+        #: without the full path to it
+        self.column_name = name
+
+        #: The type of data this attribute contains
+        self.datatype = schema.type or "string"
+
+        #: How we should associate values of this attribute to output colors
+        self.color_map = schema.extensions.get("linode-cli-color")
+
+        #: The type for items in this attribute, if this attribute is a list
+        self.item_type = None
+        if schema.type == "array":
+            self.item_type = schema.items.type
+
+    @property
+    def path(self):
+        """
+        This is a helper for filterable fields to return the json path to this
+        element in a response.
+        """
+        return self.name
+
+    def _get_value(self, model):
```

Comment:
What is a `model`? Is it a raw response?
Should we use this method to initialize the self.value as well?

Reply:
added a comment to clarify model is the JSON response. I'm not sure if we can set self.value or not its a remnant from Will's original PR. I've removed it for now.